docs/Scaling/Docker Swarm/docker-swarm.md

```diff
@@ -0,0 +1,72 @@
+--8<-- "References/abbreviations.md"
+
+### Introduction
+
+!!! important
```

Comment:
1.) There is some inconsistency in capitalization. Check out the rest of the docs. (e.g., Clone the Repository).
2.) Line 8: "This deployment uses Docker* in swarm mode. It is assumed that all the images are built and tested with `docker-compose`.  To learn more about building the images with docker-compose, please refer to  [**Express Setup**](../../Docker/dockerLocal.md). Images can only be deployed on other systems if they are pushed to a registry. 
3.) Line 14: "Before running the following commands on Linux,, confirm that the user has been added to the docker group.

Reply:
done